README.md

```diff
@@ -38,25 +44,51 @@ test.put('hello', 'world', function() {
+
 ## API
 
-### `subdb = sub(db, [prefix], [opts])`
+### `subdb = sub(db[, prefix][, options])`
+
+Returns a `levelup` instance that uses subleveldown to prefix the keys of the underlying store of `db`. The required `db` parameter must be a `levelup` instance. Any layers that this instance may have (like `encoding-down` or `subleveldown` itself) are peeled off to get to the innermost [`abstract-leveldown`][abstract-leveldown] compliant store (like `leveldown`). This ensures there is no double encoding step.
 
-Returns a `levelup` instance that uses subleveldown with `prefix` on top of the underlying *down of `db`. Each sublevel is a `levelup` of its own which can have specific encodings.
+The `prefix` must be a string. If omitted, the effective prefix is two separators, e.g. `'!!'`. If `db` is already a subleveldown-powered instance, the effective prefix is a combined prefix, e.g. `'!one!!two!'`.
 
-In other words, it's possible to have e.g. one sublevel with Buffer keys and values and another sublevel with `'utf-8'` encoded keys and `json` encoded values.
+The optional `options` parameter has the following `subleveldown` specific properties:
 
-The optional `opts` parameter has the following `subleveldown` specific properties:
+* `separator` *(string, default: `'!'`)* Character for separating sublevel prefixes from user keys and each other. Should be outside the character (or byte) range of user keys.
+* `open` *(function)* Optional open hook called when the underlying `levelup` instance has been opened. The hook receives a callback which must be called to finish opening.
 
-* `opts.separator` *(string, default: `'!'`)* Character for separating sublevels
-* `opts.open` *(function)* Optional open hook called when the underlying `levelup` instance has been opened. The hook has the signature `hook(cb)`, the callback must be called to finish opening
+Any other `options` are passed along to the underlying [`levelup`][levelup] and [`encoding-down`][encoding-down] constructors. See their documentation for further details.
 
-The `opts` argument is passed along to the underlying [`levelup`][levelup] and [`encoding-down`][encoding-down] constructors. See their documentation for further details.
+## Install
+
+With [npm](https://npmjs.org) do:
+
+```
+npm i subleveldown -S
+```
 
 ## License
 
-MIT
+MIT © 2014-present [Mathias Buus](https://github.com/mafintosh) and [contributors](https://github.com/Level/subleveldown/graphs/contributors). See the included [LICENSE](./LICENSE.md) file for more details.
```

Comment:
@ralphtheninja WDYT about doing `© <year>-present` everywhere? Means we don't have to update it.

Reply:
:100: It's painful to update this every year on all repos.